### PR TITLE
doc: fix formatting for README's setup section

### PR DESCRIPTION
**Pull request by ytvnr**
Just fix the formatting of a section of the README

## References

- TODO

## Submitter checklist

- [ ] Recommended: Join [WireMock Slack](https://slack.wiremock.org/) to get any help in `#help-contributing` or a project-specific channel like `#wiremock-java`
- [x] The PR request is well described and justified, including the body and the references
- [x] The PR title represents the desired changelog entry
- [x] The repository's code style is followed (see the contributing guide)
- [ ] Test coverage that demonstrates that the change works as expected
- [ ] For new features, there's necessary documentation in this pull request or in a subsequent PR to [wiremock.org](https://github.com/wiremock/wiremock.org)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,21 +33,21 @@ The extension also adds a Java DSL that works with the Java classes generated by
 
 Add the extension JAR dependency to your project:
 
-=== "Gradle"
+#### Gradle
```
Comment (RafeArnold) [inline README.md:36]:
```suggestion
Gradle:
```
to more accurately match what's currently on https://wiremock.org/docs/grpc/

File: README.md [hunk 2]
```diff
@@ -33,21 +33,21 @@ The extension also adds a Java DSL that works with the Java classes generated by
...
 
-    ```gradle
-    implementation 'org.wiremock:wiremock-grpc-extension:{{ versions.grpc_extension_version }}'
-    ```
+  ```gradle
+  implementation 'org.wiremock:wiremock-grpc-extension:{{ versions.grpc_extension_version }}'
+  ```
 
-=== "Maven"
+#### Maven
```
Comment (RafeArnold) [inline README.md:42]:
```suggestion
Maven:
```
to more accurately match what's currently on https://wiremock.org/docs/grpc/

**Comment by RafeArnold** [comment]
could these same changes be made to docs/index.md

File: README.md (hunk 1 above)
Comment (ytvnr) [reply README.md]: good catch! thanks :)

**Comment by ytvnr** [comment]
@RafeArnold thank you for the review, I just forced push with you suggestions and the update of docs/index.md

File: docs/index.md [hunk 3]
```diff
@@ -33,13 +33,13 @@ The extension also adds a Java DSL that works with the Java classes generated by
 
 Add the extension JAR dependency to your project:
 
-=== "Gradle"
+Gradle:
 
     ```gradle
     implementation 'org.wiremock:wiremock-grpc-extension:{{ versions.grpc_extension_version }}'
     ```
```
Comment (RafeArnold) [inline docs/index.md:40]: the indentation also needs removing

File: docs/index.md [hunk 4]
```diff
@@ -33,13 +33,13 @@ The extension also adds a Java DSL that works with the Java classes generated by
...
 
-=== "Maven"
+Maven:
 
     ```xml
     <dependency>
```
Comment (RafeArnold) [inline docs/index.md:45]: the indentation also needs removing

Comment (ytvnr) [reply docs/index.md]: done

**Review by RafeArnold — APPROVED** [review]

**Comment by RafeArnold** [comment]
thank you very much for the contribution @ytvnr !

**Pull request merged**